.codesandbox/ci.json

```diff
@@ -0,0 +1,3 @@
+{
+  "sandboxes": ["vanilla", "vanilla-ts", "new"]
```

Comment:
Can we specify a Codesandbox document to use as a template for incremental builds?

As we have a [React sandbox](https://codesandbox.io/s/goofy-darwin-xx1c8?file=/src/index.js), we may create a new sandbox dedicated for PR previews. Unlike the React one, it would use TypeScript, plain JavaScript and have a few tests. That way we could see browser and NodeJS integrations in a single sandbox.

Reply:
That's actually what I do in `rtk-query`... there is some config shenanigans we had to deal with to get the examples to run locally as well in CSB, but I could probably get both scenarios working for `msw` without too much an issue. If you look, the Svelte example uses the node template, while React uses CRA. https://github.com/rtk-incubator/rtk-query/tree/main/examples

Reply:
Just to elaborate on 'shenanigans': it was problematic getting the tests to run and to avoid errors regarding multiple versions of React. The examples would try to reference the React/jest/etc versions in the root and throw 'multiple versions of react' errors regarding hooks usage. It took a bit of tinkering to get everything configured so that things run everywhere without problems.